pkg/async/notifications/implementations/sandbox_publisher_test.go

```diff
@@ -0,0 +1,36 @@
+package implementations
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSandboxPublisher_Publish(t *testing.T) {
+	publisher := NewSandboxPublisher()
```

Comment:
Can we use NewNotificationsPublisher to create SandboxPublisher here?

Reply:
@pingsutw 
I've tried it, but the import cycle is not allowed in go test.

```
imports github.com/flyteorg/flyteadmin/pkg/async/notifications: import cycle not allowed in test
```
I've wrote code like this
```
var scope = promutils.NewScope("test_sandbox_processor")
notificationsConfig := runtimeInterfaces.NotificationsConfig{
Type: "sandbox",
}
testSandboxProcessor := notifications.NewNotificationsProcessor(notificationsConfig, scope)
```

And the dependencies I will list at here
```
package github.com/flyteorg/flyteadmin/pkg/async/notifications/implementations
imports github.com/flyteorg/flyteadmin/pkg/async/notifications: import cycle not allowed in test
```

If test the NewNotificationsPublisher  is necessary, I can try it in another package, but I am not sure whether this is what you want, thanks a lot

Reply:
I see, ok, that's fine